<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QDialog" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="PriceLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="TypeLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="FirstGraph" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="firstGraphLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="SecondGraph" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>521</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SecondGraphLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
